DigitalLearningSolutions.Web/Controllers/Signposting/SignpostingSsoController.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.Signposting
+{
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models.Signposting;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.SignpostingSso;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+
+    [FeatureGate(FeatureFlags.UseSignposting)]
+    [Route("Signposting")]
+    [Authorize(Policy = CustomPolicies.UserOnly)]
+    public class SignpostingSsoController : Controller

```

Comment:
Ah, think we just want to add an attribute to say this is in the LearningPortal DlsSubApplication

Reply:
Ah yes, done that now and it looks like this:

![image](https://user-images.githubusercontent.com/18046982/150116066-be186a04-0418-49f1-a7b9-c93362c14b6a.png)

Reply:
Beautiful!